### PR TITLE
Upload photos to google spreadsheets

### LINKED ISSUE #442: Can't send photos from collect to google spreadsheets anymore
Hi, 

I'm pretty new with the software, i'm developing a form for my employees to send field reports (with photos) and received them to my mail. I made a form, conect it to google spreadsheets and then made a script on google to send the mail. All was good and fine until I had to change some things to the form and now i receive this error everytime I want to send a picture:

whatsapp image 2017-02-20 at 3 33 38 am

I don't know what to do, this error only shows when there is a picture involved, the old version still works fine but nothing new I develop works now not even if i make a form with only a button to upload a picture. 

#### Software and hardware versions 
Collect v1.4.14, Android v5.0.2, LG Magna LG-H52g

#### Steps to reproduce the problem
just create a form on build or kobotools to send an image to a google spreadsheet. My form attached.
[IssueReport.zip](https://github.com/opendatakit/collect/files/787419/IssueReport.zip)

#### Expected behavior
received a link on a cell to the image stored in google photo, as it was before. This is a working example:
[workingExample.zip](https://github.com/opendatakit/collect/files/787422/workingExample.zip)

#### Other information 
Create new forms with only the button to send the picture, new tools to make the forms like kobotools, nothing works

Please help.

Thanks,

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Fixes #442 

I have also removed 2 duplicate gradle dependencies which were added in one of the previous PR

### CODE REVIEW

**Comment by shobhitagarwal1612** [comment]
I have pushed the commits with requested changes. 
Modified the folder directory structure as suggested by @lognaturel and @yanokwa

**Comment by grzesiek2010** [comment]
@shobhitagarwal1612 uploading looks well but I got crash when I tried to get forms from my drive:
Go to "Get Blank Form" and click on "My drive" button. Could you investigate this problem?

**Comment by shobhitagarwal1612** [comment]
The code for getting blank form is implemented in a separate PR #722

**Comment by grzesiek2010** [comment]
Ah ok I see.

File: collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java [hunk 1]
```diff
@@ -576,6 +575,8 @@ public void setTaskListener(TaskListener tl) {
             String requestString = "'" + currentDir + "' in parents and trashed=false";
             Files.List request = null;
 
+            //// TODO: 19/3/17 update library
+/*
```
Comment (nribeka) [inline collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java:579]: I'm wondering if we're gonna remove this? I'm not a big fan of commenting blocks of code 🤣

**Comment by lognaturel** [comment]
Thanks, it was helpful to look at this in isolation. No major changes to make that I can see! I'll do the full review in #722 which will be easier for you to deal with.

File: collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java (hunk 1 above)
Comment (lognaturel) [reply collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java:579]: @nribeka I asked @shobhitagarwal1612 to submit just this part of the change to take a quick look/sanity check. The rest is in #722 so let's go review there! I'm looking at it as well.

**Comment by shobhitagarwal1612** [comment]
Great :smiley:

**Pull request closed without merging**